### DISCUSSION TITLE
Area Cooling

### DISCUSSION THREAD

**Discussion by xylone** [pail23/stiebel_eltron_isg_component#248] [Q&A] [answered]
I see the two sliders, but i'm wondering what they do?
I'm talking about:
Area Cooling Flow Temperature
Area Cooling Room Temperature

I have a WPF COOL, so it does have cooling in the summer, but I don't see them in the ISG.
Can anybody explane?
Thanks!

**Comment by mbeckerkassel** [comment]
do you have an FET remote control?

**Reply by xylone** [reply]
Yes I do

**Comment by xylone** [comment, marked as answer]
I know this one now: Area Cooling Room Temperature
The temperature of the FEK needs to go above this temperature to start cooling. (Not the summer mode, but actually start cooling)

I suppose Area Cooling Flow Temperature is the target temperature of the floor water?
